.github/workflows/ruby.yml

```diff
@@ -30,7 +30,5 @@ jobs:
         run: bin/setup
       - name: Run tests
         run: bin/rake
-      - name: Run tests (with cocina feature flag enabled)
-        run: SETTINGS__FEATURES__COCINA=true bin/rake spec
       - name: Run tests (with OCFL feature flag enabled)
         run: SETTINGS__FEATURES__READ_STACKS_FROM_OCFL_ROOT=true bin/rake spec
```

Comment:
Is this one dead too?

Reply:
I'm converting that one to content-based storage.